v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-helper.ts

```diff
@@ -163,12 +163,12 @@ export class UnivariateMeasureAdornmentHelper {
       ? this.layout.getAxisMultiScale("bottom")
       : this.layout.getAxisMultiScale("left")
     const displayValue = multiScale?.formatValueForScale(value) || valueLabelString(value)
-    const plotValue = Number(displayValue) // Is this really just `value`?
+    const plotValue = Number(displayValue)
     const measureRange: IRange | undefined = attrId && this.model.hasRange
       ? this.model.computeMeasureRange(attrId, this.cellKey, dataConfig)
       : {}
     const displayRange = measureRange.min || measureRange.min === 0
```

Comment:
Not directly related to this PR, but seems like this might be better as an `isFinite()` test.

Reply:
@kswenson Does the latest state of this section seem good? I found when I used `isFinite` that I ended up with something kind of verbose, like this:

```
const displayRange = isFinite(Number(measureRange.min))
  ? multiScale?.formatValueForScale(value - Number(measureRange.min)) || valueLabelString(value - Number(measureRange.min))
  : undefined
```

After boiling that down to something more concise, I ended up with:

```
const adjustedValue = value - Number(measureRange?.min)
const displayRange = multiScale?.formatValueForScale(adjustedValue) || valueLabelString(adjustedValue)
```